packages/core/src/components/Form/Checkbox/Checkbox.js

```diff
@@ -109,6 +110,7 @@ export default class Checkbox extends React.Component<Props, State> {
                 primarySide="right"
                 rightClassName={css.labelContainer}
               />
+              {subLabel && <p className={css.subLabel}>{subLabel}</p>}
```

Comment:
the LeftRight component is responsible for displaying the label to the right of the checkbox, the sub-label should also be rendered in the right content below the main label